Makefile

```diff
@@ -1,22 +1,97 @@
-PACKAGES := $(shell glide novendor)
+PKGS := $(shell go list ./... | grep -v go.uber.org/sally/vendor)
+SRCS := $(wildcard *.go)
 
-.PHONY: install
-install:
-	glide --version || go get github.com/Masterminds/glide
+.PHONY: all
+all: test
+
+.PHONY: vendor-update
+vendor-update:
+	rm -rf vendor
+	go get -d -v -t -u -f ./...
+	go get -v github.com/Masterminds/glide
```

Comment:
The Dockerfile duplication I commented on above, I think it's fine to duplicate this as opposed to some "glide install" target. What might be better is vendoring glide itself (which needs bootstrapping, but hey).